github/orgs_actions_allowed.go

```diff
@@ -1,63 +0,0 @@
-// Copyright 2021 The go-github AUTHORS. All rights reserved.
-//
-// Use of this source code is governed by a BSD-style
-// license that can be found in the LICENSE file.
-
-package github
-
-import (
-	"context"
-	"fmt"
-)
-
-// ActionsAllowed represents selected actions that are allowed.
-//
-// GitHub API docs: https://docs.github.com/en/rest/actions/permissions
-type ActionsAllowed struct {
-	GithubOwnedAllowed *bool    `json:"github_owned_allowed,omitempty"`
-	VerifiedAllowed    *bool    `json:"verified_allowed,omitempty"`
-	PatternsAllowed    []string `json:"patterns_allowed,omitempty"`
-}
-
-func (a ActionsAllowed) String() string {
-	return Stringify(a)
-}
-
-// GetActionsAllowed gets the actions that are allowed in an organization.
-//
-// GitHub API docs: https://docs.github.com/en/rest/actions/permissions#get-allowed-actions-and-reusable-workflows-for-an-organization
-func (s *OrganizationsService) GetActionsAllowed(ctx context.Context, org string) (*ActionsAllowed, *Response, error) {
```

Comment:
My point in the last review was that this (and several other) endpoints no longer exist on this `OrganizationsService`, and that it would be better for our users to change all these moved endpoints to something like the following:

```go
// GetActionsAllowed gets the actions that are allowed in an organization.
//
// GitHub API docs: https://docs.github.com/en/rest/actions/permissions#get-allowed-actions-and-reusable-workflows-for-an-organization
// Deprecated: please use `client.Actions.GetActionsAllowed` instead.
func (s *OrganizationsService) GetActionsAllowed(ctx context.Context, org string) (*ActionsAllowed, *Response, error) {
	s2 := (*ActionsService)(s)
	return s2.GetActionsAllowed(ctx, org)
}
```

I believe this should work, but please test it out.

Reply:
That makes sense, I'll proceed with that approach then. Thanks for clarifying 👍